javascript/.eslintrc.cjs

```diff
@@ -1,12 +1,23 @@
+const stylistic = require("@stylistic/eslint-plugin");
+
+const customized = stylistic.configs.customize({
+  indent: 2,
```

Comment:
Just use the default value, if you have any idea, please say.

Reply:
indent with 2 spaced looks good to me, we can write and read more code in one line.